src/language/source.js

```diff
@@ -41,3 +41,15 @@ export class Source {
     );
   }
 }
+
+Object.defineProperty(Source, Symbol.toStringTag, {
+  get() {
+    return this.name;
+  },
+});
+
+Object.defineProperty(Source.prototype, Symbol.toStringTag, {
```

Comment:
Should this only occur in a `if (typeof Symbol === 'function' && Symbol.toStringTag)` check? Otherwise if embedded in an older node or browser context, this could throw